cvx/markowitz/builder.py

```diff
@@ -26,7 +32,7 @@ class _Problem:
     problem: cp.Problem
     model: Dict[str, Model] = field(default_factory=dict)
 
-    def update(self, **kwargs):
+    def update(self, **kwargs: Matrix) -> _Problem:
```

Comment:
Kwargs are a matrix? I thought usually a dict.

Reply:
https://adamj.eu/tech/2021/05/11/python-type-hints-args-and-kwargs/

Reply:
I didn't know that